src/oop/object_interactions/hard/q4/QuestionText.md

```diff
@@ -0,0 +1,44 @@
+# oop-object_interactions-hard-q4
+
+- Java question
+
+## Question Text
+
+Some data should be encrypted, however to do so you need a key to base the encryption off of. Design part of a `CryptBox`
+class using the provided class components, and create a `crossEncrypt` method that takes another `CryptBox` object and 
+uses that object's key to encrypt the other object's data.
+
+### CryptBox Class:
+
+Your `CryptBox` class currently has the following items:
+
+- Attributes:
+    - A `String` attribute called **lockedData**
+    - An `int` attribute called **cryptKey**
+    - Both of these attributes should not be accessible from another class
+
+- Constructor:
+    - A constructor visible from another class that accepts a `String` and an `int` parameter, and initializes the 
+      **lockedData** attribute with the `String` and the **cryptKey** attribute with the `int`
+    - This constructor should be visible from another class
+
+- Methods
+    - A `getCryptKey` method which returns the **cryptKey** attribute of the `CryptBox`
+
```

Comment:
Maybe change the visibility modifier wording of the items that are already given to "is visible" or "is not accessible" rather the "should be visible" or "should be accessible"